### PR TITLE
Farm 50 global style

### PR DESCRIPTION
**Pull request by AntoineCAU**

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — DISMISSED** [review]

**Comment by LoicBrassart** [comment]
I was reviewing when you added your last commit @AntoineCAU -_-

File: src/components/Cockpit/index.jsx [hunk 1]
```diff
@@ -28,65 +28,82 @@ function Cockpit() {
...
-                <Link to="/myapps" className="contactButtonApp">
-                  Cliquer ici pour accéder à vos applications
-                </Link>
-              )}
-              <Link to="/apps" className="contactButtonApp">
-                Cliquer ici pour accéder au catalogue d&apos;applications
+            <div className="beta">
+              <p>Farm Connect est actuellement en version &quot;BETA&quot;.</p>
+              <p>
+                N’hésitez pas à nous contacter pour nous dire ce que vous pensez
```
Comment (LoicBrassart) [inline src/components/Cockpit/index.jsx]: This didn't pass through Prettier/ESLint

File: src/components/Contact/index.jsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ export default function ContactForm() {
               <button
                 type="submit"
                 className="buttonContact"
-                disabled={handleSubmit}
+                disabled={!handleSubmit}
```
Comment (LoicBrassart) [inline src/components/Contact/index.jsx]: `handleSubmit` is a function -> it's always true

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix these plz

**Comment by Erriep** [comment]
Resolve conflits

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**